packages/core/src/client.ts

```diff
@@ -2206,6 +2206,11 @@ export class MedplumClient extends EventTarget {
       return undefined as unknown as T;
     }
 
+    if (response.status === 404) {
+      console.error(`404, couldn't retrieve response from ${url}`);
+      throw new OperationOutcomeError(normalizeOperationOutcome(notFound));
```

Comment:
Does this error being thrown also result in any output?

Reply:
I could be wrong but I believe the 404 output will result in a notFound

Reply:
@mattwiller to your point, it seems like we do handle all responses >= 400 on line 2222.

@codyebberson   - do you remember why we handle 400's after the `.json()` parse?